### PR TITLE
feat: use post reviews data from redux

### PR DESCRIPTION
**Pull request by liruchen32**
# Issue
[[會員制]重要報導回顧-串接](https://app.asana.com/0/1203699264568808/1206721264374718/f)

# Notice

# Dependency
[npm-packges#533](https://github.com/twreporter/twreporter-npm-packages/pull/533)

### CODE REVIEW

File: src/components/my-reading/reviewing-article-section.js [hunk 1]
```diff
@@ -81,15 +76,35 @@ const ReviewingArticleSection = () => {
...
-    }
-    setTimeout(() => setIsLoading(false), 1000)
+    const jwt = _.get(state, [reduxStateFields.auth, 'accessToken'])
+    dispatch(getPostReviews(jwt)).then(res => {
+      const reviews = _.get(res, ['payload', 'data', 'data'], [])
+      const formattedReviews = reviews.map(review => {
+        const { slug, reviewWord, title } = review
+        const { url: bgImage } = _.get(
+          review,
+          ['og_image', 'resized_targets', 'mobile'],
```
Comment (Aylie-Chou) [inline src/components/my-reading/reviewing-article-section.js:86]: mobile -> w400 ?

**Review by Aylie-Chou — CHANGES_REQUESTED** [review]
as comment ~

File: src/components/my-reading/reviewing-article-section.js (hunk 1 above)
Comment (liruchen32) [reply src/components/my-reading/reviewing-article-section.js]: fixed

**Review by Aylie-Chou — APPROVED** [review]
LGTM

**Pull request merged**